aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -1068,6 +1064,7 @@ ReduceConfig setReduceConfig(const TensorIterator& iter){
       // Note that if vt0 < ReduceConfig::vec_size, then this means the register pressure could be high, in such case,
       // we should avoid vectorization.
       config.vectorize_input = true;
+      dim0 /= config.input_vec_size;
```

Comment:
This is the only relevant change of the PR, the rest are pretty much cosmetic changes.